### PR TITLE
Fix many Android lint warnings/errors

### PR DESCRIPTION
**Pull request by jverbraeken**
- Relevant Issues: -
- Related Pull Requests: -

* * *

## What
This Pull Request adds to the repository fixes more many Android Lint warnings/errors

## Why
This Pull Request is needed because there were quite some warnings/errors and most of these were relevant and indicated small code flaws in our code

## How
This feature can be viewed/tested within the project by running Android Lint

## Alternative implementation
Other implementations that I've have considered are 

## Notes

The only two warnings that I didn't fix were a warning that Eric's visual verification image is not available in mdpi, xhdpi and xxhdpi and that labelFor was missing for a file, but both are not easy to fix and are just a small warning (2/10)

### CODE REVIEW

**Comment by ericcornelissen** [comment]
I started on something related to this on [order-strings](https://github.com/ericcornelissen/NervousFish/tree/order-strings) 😄 Are you OK if I merge that branch in this PR?

**Comment by jverbraeken** [comment]
@ericcornelissen 

> I started on something related to this on order-strings 😄 Are you OK if I merge that branch in this PR?

No problem 👍

**Comment by ericcornelissen** [comment]
As the title of the branch ("order strings") suggests, its not aimed at fixing any actual errors, it is just ordering the strings defined in `../values/strings.xml` and `../values-nl/strings.xml` to make them easier to read for humans 😄

File: app/src/main/res/values-nl/strings.xml [hunk 1]
```diff
@@ -31,6 +31,7 @@
     <string name="go_back">Ga terug</string>
     <string name="invalid_name">Ongeldige naam</string>
     <string name="no_devices_found">Geen apparaten gevonden…</string>
+    <string name="remember_my_choice">Onthoud mijn keuze</string>
```
Comment (clenrock) [inline app/src/main/res/values-nl/strings.xml]: I think this should be 'onthoudt'

File: app/src/main/res/values-nl/strings.xml [hunk 2]
```diff
@@ -77,4 +78,11 @@
     <string name="activating_bluetooth_location_text">Je moet toestemming geven om je locatie te delen om Bluetooth the kunnen gebruiken. We zullen helemaal niets met je locatie doen.</string>
     <string name="activating_bluetooth_went_wrong">Bluetooth kon niet worden geactiveerd</string>
     <string name="activating_bluetooth_went_wrong_explanation">Zet Bluetooth aan als je keys wilt versturen over Bluetooth</string>
+
+    <!-- Kies verificatie methode -->
+    <string name="select_verification_method">Selecteer een verificatie methode</string>
+    <string name="select_visual_verification">Visueel patroon</string>
+    <string name="select_rhythm_verification">Tab een ritme</string>
```
Comment (clenrock) [inline app/src/main/res/values-nl/strings.xml]: I think this should be 'tap'

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -77,4 +77,11 @@
     <string name="activating_bluetooth_went_wrong_explanation">Please enable Bluetooth if you want to exchange keys over Bluetooth.</string>
     <string name="activating_bluetooth">Activating Bluetooth…</string>
     <string name="activating_bluetooth_location_text">Accepting to share your location is something needed to run Bluetooth, note that we won\'t do anything with your location.</string>
+
+    <!-- Select verification method -->
+    <string name="select_verification_method">Select a verification method</string>
+    <string name="select_visual_verification">Visual pattern</string>
+    <string name="select_rhythm_verification">Tab a rhythm</string>
```
Comment (clenrock) [inline app/src/main/res/values/strings.xml]: This should be tab with a p

File: app/src/main/res/values-nl/strings.xml (hunk 1 above)
Comment (ericcornelissen) [inline app/src/main/res/values-nl/strings.xml]: I don't think so 😄

**Pull request merged**